gr-blocks/examples/msg_passing/hier/test_msg_hier.grc

```diff
@@ -24,6 +24,9 @@ options:
     title: ''
     window_size: 1280, 1024
   states:
+    bus_sink: false
```

Comment:
It looks like these 3 entries here and elsewhere come from using a recent GRC, and that these are defaults ... yes? If so, what I'd greatly prefer is to do these updates in 1 commit, then the msg port id updates in another commit. They are different, so let's keep them separate.

Reply:
I used gnuradio-companion from `maint-3.8`, and this is for the `maint-3.8` branch so I don't see why they shall not be included here. I don't think it makes sense to split them out in a separate commit in this case.